oxidation/client/src/plugins/libparsec/index.ts

```diff
@@ -9,12 +9,8 @@ export const libparsec = registerPlugin<LibParsecPlugin>(
   'LibParsec',
   {
     web: async () => {
-      // Don't put the module name as a literal in the import function,
-      // otherwise the compilation will eagerly try to access this module
-      // which doesn't exists when building from scratch for non-web.
-      const moduleName = '../../../pkg';
-      const libparsecWasm: any = await import(moduleName);
-      await libparsecWasm.default();  // Call wasm module's init
+      const libparsecWasm = await import('../../../pkg');
```

Comment:
Because you have to build the `pkg`, use the `const` in import has no effect, that's why `web` didn't work

Reply:
I don't understand your answer, can you be more specific ?

> Because you have to build the pkg

1) When building for electron, `pkg` doesn't get build and it's normal
2) When building for electron with `pkg` existing (because a bould for web has be done previously), this `pkg` gets added to the release. This is something we don't want given this `pkg` is a web-specific stuff that won't be used in electron/android

The code you removed was a hack to avoid this by preventing the js build pipeline (not sure which component does that 🤔 ) from considering pkg is always a dependency to add to the project.

Reply:
Ok I see what you mean

Reply:
I have no idea to solve the problem then

Reply:
@NicoTuxx @Landeers do you know a way to fix this ?

Reply:
Maybe the problem is web, it try to import `<url>/../../../pkg` and on electron the assets archive contains a path `../../../pkg` :thinking:

Reply:
When the transpiler see import('literal'), it try to load the resources, so yes it imports `<url>/blabla` and loaded before.
And when there is only a variable, it doesn't know that it has to load these resources